homeassistant/components/xiaomi_miio/sensor.py

```diff
@@ -658,6 +661,33 @@ def __init__(self, name, device, entry, unique_id, coordinator, description):
         self._attr_native_value = self._determine_native_value()
         self._attr_extra_state_attributes = self._extract_attributes(coordinator.data)
 
+    def _extract_value_from_attribute(self, state, attribute):
```

Comment:
why not just override `_parse_time_delta` instead of this whole method?

Reply:
Good catch, done.